freqtrade/freqtradebot.py

```diff
@@ -327,11 +341,11 @@ def _get_trade_stake_amount(self) -> Optional[float]:
         if avaliable_amount < stake_amount:
             raise DependencyException(
                 'Available balance(%f %s) is lower than stake amount(%f %s)' % (
-                    avaliable_amount, self.config['stake_currency'],
-                    stake_amount, self.config['stake_currency'])
+                    float(avaliable_amount), self.config['stake_currency'],
```

Comment:
isn't `get_balance` returning a float anyway ... )(and `stake_amount` should be float too?